universe/assets_3d/api_assets_3d.go

```diff
@@ -393,3 +394,61 @@ func (a *Assets3d) apiRemoveAsset3dByID(c *gin.Context) {
 
 	c.JSON(http.StatusOK, nil)
 }
+
+// @Summary Update 3d asset meta by its id
+// @Schemes
+// @Description Update 3d asset meta by its id
+// @Tags assets3d
+// @Accept json
+// @Produce json
+// @Param body body assets_3d.apiUpdateAsset3dByID.InBody true "body params"
+// @Success 200 {object} assets_3d.apiUpdateAsset3dByID.Out
+// @Failure 400 {object} api.HTTPError
+// @Failure 500 {object} api.HTTPError
+// @Router /api/v4/assets-3d/{space_id}/{asset3d_id} [patch]
+func (a *Assets3d) apiUpdateAsset3dByID(c *gin.Context) {
+	asset3dID, err := uuid.Parse(c.Param("asset3dID"))
+	if err != nil {
+		err = errors.WithMessage(err, "Assets3d: apiUpdateAsset3dByID: failed to parse uuid")
+		api.AbortRequest(c, http.StatusInternalServerError, "invalid_uuid_parse", err, a.log)
+		return
+	}
+
+	type InBody struct {
```

Comment:
better to move the inBody binding to the top to keep it consistent with other endpoints

Reply:
Moved